ModularTegustation/lc13_machinery.dm

```diff
@@ -0,0 +1,109 @@
+//Links to abnormality consoles when the console spawns
+/obj/machinery/containment_panel
+	name = "containment_panel"
+	desc = null
+	icon = 'ModularTegustation/Teguicons/lc13doorpanels.dmi'
+	icon_state = "frame_control"
+	density = FALSE
+	use_power = 0
+	var/department = "control"
+	var/linked_console
+	var/work
+	var/turf/consoletarget //for post spawn editing
+
+/obj/machinery/containment_panel/Initialize()
+	consoletarget = locate((x + 1), (y + 3), z) //always left of the containment door
+	RegisterSignal(SSdcs, COMSIG_GLOB_ABNORMALITY_SPAWN, .proc/locate_console)
+	. = ..()
+
+/obj/machinery/containment_panel/process()
```

Comment:
I'd avoid using process on your place.
It can be avoided entirely with the use of signals.

Reply:
I tried using signals it didnt work.

Reply:
Im trying to make it work with signals it still doesnt work